tests/core/test_farmer_harvester_rpc.py

```diff
@@ -435,7 +435,7 @@ def test_plot_matches_filter(filter_item: FilterItem, match: bool):
         ),
         (FarmerRpcClient.get_harvester_plots_invalid, [], None, True, 13),
         (FarmerRpcClient.get_harvester_plots_invalid, ["invalid_0"], None, False, 6),
-        (FarmerRpcClient.get_harvester_plots_invalid, ["inval", "lid_1/"], None, False, 2),
+        (FarmerRpcClient.get_harvester_plots_invalid, ["inval", "lid_1"], None, False, 2),
```

Comment:
@xdustinface, is the need to remove the `/` here a sign of some lack of normalization of the paths or something?  @emlowe, could you comment on how this was failing?  Was it `/` vs. `\`?  or just no trailing slash to match at all?

Reply:
Yea, it's a string \ vs / - I dunno if there is a better way to build the test, but removing the / was what Kevin suggested

Reply:
I'm definitely tempted to normalize them for comparison, though I haven't thought through corner cases.  `/` is generally valid all over Windows.  But, that would be a feature addition so not really required here.  But, might be worth xfailing this on Windows?  This suggestion needs at least an `import sys` added so apply this manually.

pytest.param(1, 3, marks=pytest.mark.xfail(reason="some bug")),
```suggestion
        pytest.param(
            FarmerRpcClient.get_harvester_plots_invalid,
            ["inval", "lid_1/"],
            None,
            False,
            2,
            marks=pytest.mark.xfail(condition=sys.platform == "win32", reason="not presently supported on Windows"),
        ),
        pytest.param(
            FarmerRpcClient.get_harvester_plots_invalid,
            ["inval", "lid_1\\"],
            None,
            False,
            2,
            marks=pytest.mark.skip(condition=sys.platform != "win32", reason="only for Windows"),
        ),
```

Reply:
@altendky It's good enough to just drop the `/`. It's just filtering for the path there, not really important to test that path filtering for windows is with `\` and for others with `/` imo. I initially just added it to avoid flakiness with the temp directory names because i thought it might contain `lid_1` but well let's see how that goes?